pandas/tests/test_groupby.py

```diff
@@ -6391,6 +6391,14 @@ def testit(label_list, shape):
     testit(label_list, shape)
 
 
+def test_nunique_empty_Series():
+    series = Series()
```

Comment:
try with ``s = Series(name='foo')``

use ``tm.assert_series_equal`` for the comparison